packages/augur-ui/src/modules/markets-list/components/markets-header.styles.less

```diff
@@ -82,11 +83,13 @@
     }
 
     > div {
-      display: flex;
+      display: none;
       align-items: center;
 
       @media @breakpoint-mobile {
-        flex-direction: column;
+        display: flex;
+        flex-direction: row;
```

Comment:
minor thing, but since the flex-direction is row by default I prefer we don't specify.